### ISSUE TITLE
Audio Recorder doesn't work in Moodle with Firefox

### ISSUE THREAD

**Issue by Urpokarhu1** [h5p/h5p-audio-recorder#81]
Hello,
Audio recorder doesn't in Moodle with Firefox when you press "record" it gives an error "Access to microphone is not allowed in your browser since this page is not served using HTTPS. Please contact the author, and ask him to make this available using HTTPS"

Moodle 4.1 & mod_hvp 1.26.1

![image](https://github.com/h5p/h5p-audio-recorder/assets/53904029/44880145-61c7-4d07-a70b-af5e869aa456)

**Comment by thomasmars** [comment]
Is your page served using HTTPS ? Does it work in other browsers ?

**Comment by Urpokarhu1** [comment]
Yes my site is using HTTPS, it does work with Chrome but not with Firefox. Also this works with Firefox too https://h5p.org/audio-recorder only when the recorder is used inside Moodle the problem arises.
